java/examples/README.md

```diff
@@ -1,8 +1,55 @@
-Building and Running
---------------------
+# CuVS Java API Examples
 
-Make sure to have JDK 22 and Maven 3.9.6+.
+This maven project contains simple examples for CAGRA, HNSW, and Bruteforce algorithms.
 
-    mvn clean compile assembly:single
+## Prerequisites
+- All the prerequisites in the CuVS Java API readme
```

Comment:
This would be where you'd point to the cuVS build and install guide, which lists the dependencies for libcuvs.